drivers/serial/Kconfig.nrfx

```diff
@@ -17,53 +17,61 @@ menuconfig UART_NRF5
 	  family processors. Note that there is only one
 	  UART controller on nRF5 ICs.
 
-config UART_NRF5_CLK_FREQ
+config UART_NRFX_UART_CLK_FREQ
 	int
 	default 16000000
-	depends on UART_NRF5
+	depends on UART_NRFX_UART
 
-config UART_NRF5_FLOW_CONTROL
+config UART_NRFX_UART_PARITY_BIT
+        bool "Parity bit enabled"
+        default n
+        depends on UART_NRFX_UART
+        help
+          Enable parity bit.
+
+config UART_NRFX_UART_FLOW_CONTROL
 	bool "Enable Flow Control"
 	default n
-	depends on UART_NRF5
+	depends on UART_NRFX_UART
 	help
 	  Enable Flow Control. If selected, additionally two pins, RTS and CTS
 	  has to be configured.
 
-config UART_NRF5_GPIO_TX_PIN
+config UART_NRFX_UART_GPIO_TX_PIN
 	int "TX Pin Number"
 	range 0 47 if SOC_NRF52840_QIAA
 	range 0 31
-	depends on UART_NRF5
+	depends on UART_NRFX_UART
 	depends on GPIO_NRF5
```

Comment:
This is unnecessary. The `_TX_PIN` option only allows to select a pin number, it does not need GPIO_NRF5 to do so.
This applies to the other pin configs here as well.